webv2/api/schema_test.go

```diff
@@ -258,7 +258,7 @@ func TestGetDDL(t *testing.T) {
 						"c2": ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Int64}, NotNull: true},
 						"c3": ddl.ColumnDef{Name: "c", T: ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, NotNull: true}},
 					PrimaryKeys: []ddl.IndexKey{ddl.IndexKey{ColId: "c1", Desc: false}},
-					ForeignKeys: []ddl.Foreignkey{ddl.Foreignkey{Name: "fk1", ColIds: []string{"c1"}, ReferTableId: "t2", ReferColumnIds: []string{"c4"}, OnDelete: constants.CASCADE, OnUpdate: constants.NO_ACTION}},
+					ForeignKeys: []ddl.Foreignkey{ddl.Foreignkey{Name: "fk1", ColIds: []string{"c1"}, ReferTableId: "t2", ReferColumnIds: []string{"c4"}, OnDelete: constants.FK_CASCADE, OnUpdate: constants.FK_NO_ACTION}},
```

Comment:
I don't think we seem to be handling the case where the foreign key from the table gets dropped, in that case the warning will also need to be dropped if its the only foreign key with FK actions

Reply:
Was planning to handle this with the interleaving bit as it directly calls DropForeignKey - is that okay?

Reply:
For the UI part, the warning are dropped from the tab and the count is also lowered, just that internally the issue list might have duplicate issues; had discussed this with deep and he said to de-prioritize it so kept it for the end

Reply:
okay sure, sounds good